recipes/java-jdk/meta.yaml

```diff
@@ -0,0 +1,39 @@
+# Original recipe: birdhouse channel
+
+{% set name = "java-jdk" %}
+{% set version = "8.0.112" %}
+{% set zulu_build = "8.19.0.1" %}
+{% set md5_linux = "3f95d82bf8ece272497ae2d3c5b56c3b" %}
+{% set md5_osx = "8d9cce47fbc8f80dafbfc88fa4ae5d07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ zulu_build }}-{{ version }}.tar.gz  # [linux]
+  fn: {{ name }}-{{ zulu_build }}-{{ version }}.zip  # [osx]
+  url: http://cdn.azul.com/zulu/bin/zulu{{ zulu_build }}-jdk{{ version }}-linux_x64.tar.gz  # [linux]
+  url: http://cdn.azul.com/zulu/bin/zulu{{ zulu_build }}-jdk{{ version }}-macosx_x64.zip  # [osx]
+  md5: {{ md5_linux }}  # [linux]
+  md5: {{ md5_osx }}  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+  string: zulu{{ zulu_build }}_{{PKG_BUILDNUM}}
```

Comment:
Shouldn't this have a variant/features set to zulu, like with blas / openblas / mkl /etc.